performant-software/dm-2
# Open targeted document next to origin

### What this PR does

- Per #323:
  - Opens targeted document directly to the right of/below the origin document (by instance key)

### Status


### Steps to test

1. Visit https://dm-2-staging.herokuapp.com/
2. Log in
3. Open or create a project
4. Open or create several documents, including at least one with highlights linking to other documents
5. In an open document with highlights, click on one of the highlights
6. Follow a link in that highlight and verify that the target document opens directly to the right of, or directly below, the origin document
7. Verify that 6 still works even when additional documents are open to the right of/below the origin document
